manifests/vhost.pp

```diff
@@ -263,8 +267,12 @@
 
   # Same as above, but for logroot
   if ! defined(File[$logroot]) {
+    $logroot_ensure_real = $ensure? {
+      present => $logroot_ensure,
+      default => 'absent',
+    }
     file { $logroot:
-      ensure  => $logroot_ensure,
+      ensure  => $logroot_ensure_real,
```

Comment:
I'm not really comfortable changing the default behavior for logroot, given the possible security implications of the change (security folks I have worked with were very particular about not having logs removed, and this could accidentally break that. I think it's best to require people to be explicit)